.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: release
+# Trigger on release creation
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      RELEASE_ZIP_NAME: flybywiresim-a32nx.zip
+      MOD_FOLDER_NAME: A32NX
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+      - name: Setup Python 3
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
```

Comment:
This step isn't needed anymore and can safely be removed.

Reply:
Yes that's right, just removed it!